tests/foreman/cli/test_host.py

```diff
@@ -70,1769 +64,1491 @@
+            'puppet-proxy-id': module_default_proxy['id'],
+            'root-password': host_template.root_pass,
+        }
+    )
+
+
+@pytest.fixture(scope="function")
+def function_make_user(function_make_host):
+    """
+    Returns dict with user object and with password to this user
+    """
+    user_name = gen_string('alphanumeric')
+    user_password = gen_string('alphanumeric')
+    org = Org.info({'name': function_make_host['organization']})
+    location = Location.info({'name': function_make_host['location']})
+    user = make_user(
+        {
+            'admin': False,
+            'default-organization-id': org['id'],
+            'organization-ids': [org['id']],
+            'default-location-id': location['id'],
+            'location-ids': [location['id']],
+            'login': user_name,
+            'password': user_password,
+        }
+    )
+    return {'user': user, 'password': user_password}
+
+
+# -------------------------- CREATE SCENARIOS -------------------------
+@pytest.mark.host_create
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_create_and_delete(module_lce_library, module_published_cv):
+    """A host can be created and deleted
+
+    :id: 59fcbe50-9c6b-4c3c-87b3-272b4b584fb3
+
+    :expectedresults: A host is created and deleted
+
+    :BZ: 1260697, 1313056, 1361309
+
+    :CaseImportance: Critical
+    """
+    name = valid_hosts_list()[0]
+    host = entities.Host()
+    host.create_missing()
+    interface = (
+        'type=interface,mac={},identifier=eth0,name={},domain_id={},'
+        'ip={},primary=true,provision=true'
+    ).format(host.mac, gen_string('alpha'), host.domain.id, gen_ipaddr())
+    new_host = make_host(
+        {
+            'architecture-id': host.architecture.id,
+            'content-view-id': module_published_cv.id,
+            'domain-id': host.domain.id,
+            'environment-id': host.environment.id,
+            'interface': interface,
+            'lifecycle-environment-id': module_lce_library.id,
+            'location-id': host.location.id,
+            'mac': host.mac,
+            'medium-id': host.medium.id,
+            'name': name,
+            'operatingsystem-id': host.operatingsystem.id,
+            'organization-id': host.organization.id,
+            'partition-table-id': host.ptable.id,
+            'root-password': host.root_pass,
+        }
+    )
+    assert f'{name}.{host.domain.read().name}' == new_host['name']
+    assert new_host['organization'] == host.organization.name
+    assert new_host['content-information']['content-view']['name'] == module_published_cv.name
+    assert (
+        new_host['content-information']['lifecycle-environment']['name'] == module_lce_library.name
+    )
+    host_interface = HostInterface.info(
+        {'host-id': new_host['id'], 'id': new_host['network-interfaces'][0]['id']}
+    )
+    assert host_interface['domain'] == host.domain.read().name
+
```

Comment:
I would add some test-level log message here saying, that we proceeded to the next stage of the test (level: info)
(suggestion, not a blocker)